src/renderer/containers/layout/standardLeftDrawer/standardLeftDrawerClass.jsx

```diff
@@ -33,24 +36,10 @@ export default class StandardLeftDrawer extends React.Component {
             checked={itemChecked}
             label={child.description}
             onChange={value => {
-              this.props.getServiceLocations(child.id, value);
-              let taxSpread = this.props.selectedServices[taxId]
-                ? this.props.selectedServices[taxId]
-                : {};
-              if (!value) {
-                // unchecked, so omit from array.
-                taxSpread = omit([child.id], taxSpread);
-              } else {
-                taxSpread = {
-                  ...taxSpread,
-                  [child.id]: value,
-                };
-              }
-              const updateData = {
-                ...this.props.selectedServices,
-                [taxId]: taxSpread,
-              };
-              this.props.set(updateData, selectedServices);
+              this.props.set(
+                this._getServices(child, value, this.props.selectedServices),
+                selectedServices
```

Comment:
might be beneficial to alias this as `selectedServicesState` or `selectedServicesReduxState` so that it is clear which one is which. Easier to differentiate between props.selectedServices and selectedServices path.